streams/src/main/java/org/apache/kafka/streams/processor/StateStore.java

```diff
@@ -58,4 +58,10 @@
      * @return  {@code true} if the storage is persistent&mdash;{@code false} otherwise
      */
     boolean persistent();
+
+    /**
+     * Is this store open
+     * @return {@code true} if the store is open
+     */
+    boolean isOpen();
```

Comment:
What's the meaning of being open (`isOpen() == true`)?  Does it mean the store is ready for writing/reading?